docs/mkdocs/tutorials/projects.md

```diff
@@ -0,0 +1,131 @@
+# Creating & Retrieving Projects
+
+## Creating new projects
+
+New projects for new data can be created in the following way:
+
+Source: `examples/gRPC/meta/gRPC_pb_createProject.py`
+
+```python
+import os
+import sys
+
+import meta_operations_pb2_grpc as metaOperations
+import projectCreation_pb2
+
+script_dir = os.path.dirname(__file__)
+util_dir = os.path.join(script_dir, '..')
+sys.path.append(util_dir)
+import util
```

Comment:
In this tutorial "util" doesn't exist. Maybe link to the file? Or use a minimal version of "util.get_gRPC_channel" directly in this code?

Reply:
If I get that correct, you mean that the `util.py` is never referenced in the docs?. Because then I would like to include it in the overview section, to keep the scripts as straight forward as possible.

Reply:
Kind of. The problem is that you can not just copy paste the script. If you try to execute it it'll throw the error that it cannot find util.
(always given that the seerep-msgs and seerep-com are installed, but in the long run the installation should be possible using apt)